### ISSUE TITLE
TGIsIsomorphic crash 2 

### ISSUE THREAD

**Issue by eobr007** [thetensor-space/TameGenus#14]
```
> G:Magma;
PCGroup(\[ 6, -3, 3, 3, 3, -3, 3, 1981, 8912, 1034, 873, 735 ])
> H:Magma;
PCGroup(\[ 6, -3, 3, 3, 3, -3, 3, 2953, 8912, 3950, 6483, 873, 735 ])
> TGIsIsomorphic (G, H);

TGIsIsomorphic(
    G: GrpPC : G,
    H: GrpPC : H
)
In file "/home/eobr007/Stabiliser/TameGenus/src/Iso.m", line 416, column 41:
>>       im := [(V!((x @ phi_G @ maps_G[1])*X)) @@ maps_H[1] @@ phi_H : x in G
                                           ^
Runtime error: Bad argument types

debug> 
```

**Comment by joshmaglione** [comment]
Thanks @eobr007. I am extracting the code to reproduce it.
```
G := PCGroup(\[ 6, -3, 3, 3, 3, -3, 3, 1981, 8912, 1034, 873, 735 ]);
H := PCGroup(\[ 6, -3, 3, 3, 3, -3, 3, 2953, 8912, 3950, 6483, 873, 735 ]);
TGIsIsomorphic(G, H);
```

**Comment by joshmaglione** [comment]
Looks like this is issue is similar to Issue #13, and it was fixed with commit 9d57d59.
